hivemq-edge/src/frontend/src/extensions/datahub/components/controls/DesignerToolbox.tsx

```diff
@@ -138,7 +142,15 @@ const DesignerToolbox: FC = () => {
                     <>
                       <Box pt={5} h="100%">
                         {activeStep === 0 && <ToolboxNodes />}
-                        {activeStep === 1 && <ToolboxDryRun onActiveStep={onActiveStep} />}
+                        {activeStep === 1 && (
```

Comment:
suggestion: Instead of using _magic numbers_ for the steps, using constants named accordingly to describe what each step is would make the code easier to follow.

Plus, if steps were to change, it would be less prone to errors.